ml/write_dataset_config.py

```diff
@@ -274,42 +282,47 @@ def estimationMethodAndClassMapGenerator():
             ],
             "cut_string": (estimation.get_cuts() + channel_qcd.cuts + additional_cuts).expand(),
             "weight_string": estimation.get_weights().extract(),
-            "class":classes_map[estimation.name]
+            "class": classes_map[estimation.name]
         }
-    else: ## ff and not em
+    else:  ## ff and not em
         estimation = DataEstimation(era, args.base_path, channel)
         estimation.name = "ff"
         aiso = copy.deepcopy(channel)
-        if args.channel in ["et","mt"]:
-            aisoCut=Cut("byTightIsolationMVArun2017v2DBoldDMwLT2017_2<0.5&&byVLooseIsolationMVArun2017v2DBoldDMwLT2017_2>0.5", "tau_aiso")
-            fakeWeightstring="ff2_nom"
+        if args.channel in ["et", "mt"]:
+            aisoCut = Cut(
+                "byTightIsolationMVArun2017v2DBoldDMwLT2017_2<0.5&&byVLooseIsolationMVArun2017v2DBoldDMwLT2017_2>0.5",
```

Comment:
These will have to be switched to deeptau, but not in this pr i guess

Reply:
Yeah, the changes in dataset_config are not really from me. Someone with knowledge on this should do it.

Reply:
@mova i guess

Reply:
this is fine for now, we can change this after the PR

Reply:
I will change this when the samples are available.